File: 6. Python/EX33.py
# Inicializa variáveis
som = 0
aux = 0

# Solicita dois números
print("# Digite dois numeros:\n")

# Lê os números
n1 = int(input("# Digite N1: "))
n2 = int(input("# Digite N2: "))

# Ordena n1 e n2
if n1 > n2:
    aux = n1    # Armazena n1
    n1 = n2     # Troca n1 por n2
    n2 = aux    # Atribui aux a n2

# Soma ímpares entre n1 e n2
for i in range(n1 + 1, n2):
    if i % 2 != 0:      # Verifica ímpar
        som += i        # Adiciona à soma

# Exibe a soma
print(f"# Soma dos Impares = {som}")

File: 4. Java/EX33.java
package Apostila;

import java.util.Locale;
import java.util.Scanner;

public class EX33
{
    public static void main(String[] args)
    {
        int n1, n2;
        int som = 0, aux = 0;

        Locale.setDefault(Locale.US);

        try (Scanner ent = new Scanner(System.in))
        {
            System.out.println("# Digite dois numeros: ");

            System.out.print("# Digite N1: ");
            n1 = ent.nextInt();

            System.out.print("# Digite N2: ");
            n2 = ent.nextInt();
        }

        if (n1 > n2)
        {
            aux = n2;
            n2 = n1;
            n1 = aux;
        }

        for (int i = n1 + 1; i < n2; i++)
        {
            if (i % 2 != 0)
            {
                som += i;
            }
        }

        System.out.println("# Soma dos Impares = " + som);        
    }
}